staubli_rx160_support/urdf/rx160l.urdf

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" ?>
```

Comment:
Please regenerate this urdf using `rosrun xacro xacro -o rx160l.urdf rx160l.xacro`. That should get rid of the comments (`RAL 1028` etc), as well as the commented lines.